src/compas_ghpython/components/Compas_Frame/code.py

```diff
@@ -0,0 +1,21 @@
+"""
+Draws a frome or plane.
+"""
+from ghpythonlib.componentbase import executingcomponent as component
+
+from compas_ghpython.artists import FrameArtist
+from compas_ghpython.components import coerce_frame
+
+
+class CompasFrame(component):
+    def RunScript(self, plane):
+        origin = None
+        axes = []
+
+        if plane:
+            frame = coerce_frame(plane)
+            artist = FrameArtist(frame)
+
+            origin, axes = artist.draw()
```

Comment:
As mentioned on Slack, I believe what the artist does is not correct. To put it in words, I believe it returns non-semantic geometry: lines and a point, but in Grasshopper, we could return `Rhino.Geometry.Plane` object instead and that will be natively drawn and interpreted as a plane.

Before merging this, I would vote for changing the artist (maybe in this very PR) to that behavior. The behavior of returning lines and an origin could move to two additional methods of the artist: `.draw_origin()` and `.draw_axes()`.

@brgcode what do you think?

Reply:
not sure. i find it a bit confusing that frame visualisation outputs a plane, but i guess Rhino doesn't make the distinction and doesn't provide an actual frame object so i guess this is the best we can do...

Reply:
perhaps a note or warning can be added to the docstring?